functions-and-operators/window-functions.md

```diff
@@ -8,22 +8,447 @@ aliases: ['/docs/dev/functions-and-operators/window-functions/','/docs/dev/refer
+
+The `NTILE(n)` function divides the window in `n` groups and returns the number of the group.
+
+```sql
+WITH RECURSIVE cte(n) AS (
+    SELECT
+        1
+    UNION
+    SELECT
+        n+1
+    FROM
+        cte
+    WHERE
+    n<10
+)
+SELECT
+    n,
+    NTILE(5) OVER (ORDER BY n),
+    NTILE(2) OVER (ORDER BY n)
+FROM
+    cte;
+```
+
+```
++------+----------------------------+----------------------------+
+| n    | NTILE(5) OVER (ORDER BY n) | NTILE(2) OVER (ORDER BY n) |
++------+----------------------------+----------------------------+
+|    1 |                          1 |                          1 |
+|    2 |                          1 |                          1 |
+|    3 |                          2 |                          1 |
+|    4 |                          2 |                          1 |
+|    5 |                          3 |                          1 |
+|    6 |                          3 |                          2 |
+|    7 |                          4 |                          2 |
+|    8 |                          4 |                          2 |
+|    9 |                          5 |                          2 |
+|   10 |                          5 |                          2 |
++------+----------------------------+----------------------------+
+10 rows in set (0.00 sec)
+```
+
+## [`PERCENT_RANK()`](https://dev.mysql.com/doc/refman/8.0/en/window-function-descriptions.html#function_percent-rank)
+
+The `PERCENT_RANK()` function returns a number between 0 and 1 indicating the percentage of rows with a value less than the current window.
+
+```sql
+SELECT 
+    *,
+    PERCENT_RANK() OVER (ORDER BY n),
+    PERCENT_RANK() OVER (ORDER BY n DESC)
+FROM (
+    SELECT 5 AS 'n'
+    UNION ALL
+    SELECT 8
+    UNION ALL
+    SELECT 5
+    UNION ALL
+    SELECT 30
+    UNION ALL
+    SELECT 31
+    UNION ALL
+    SELECT 32) a;
+```
+
+```
++----+----------------------------------+---------------------------------------+
+| n  | PERCENT_RANK() OVER (ORDER BY n) | PERCENT_RANK() OVER (ORDER BY n DESC) |
++----+----------------------------------+---------------------------------------+
+|  5 |                                0 |                                   0.8 |
+|  5 |                                0 |                                   0.8 |
+|  8 |                              0.4 |                                   0.6 |
+| 30 |                              0.6 |                                   0.4 |
+| 31 |                              0.8 |                                   0.2 |
+| 32 |                                1 |                                     0 |
++----+----------------------------------+---------------------------------------+
+6 rows in set (0.00 sec)
+```
+
+## [`RANK()`](https://dev.mysql.com/doc/refman/8.0/en/window-function-descriptions.html#function_rank)
+
+The `RANK()` function is similar to [`DENSE_RANK()`](#dense_rank) but will leave any gaps in case of ties.
```

Comment:
Maybe also mention that is means absolute ranking, i.e. also taking ties into account / rank of 7 means there are 6 rows with lower rank?